packages/sdk/tests/test-utils.ts

```diff
@@ -170,314 +138,25 @@ export const timeouts = {
   tx: 2 * MAX_EXPECTED_BLOCKTIME,
 };
 
-export function getOrderObject({
```

Comment:
Method renamed to `getDatasetOrderObject()` and moved to `utils/datasetOrders.ts`